docs/windows.md

```diff
@@ -0,0 +1,64 @@
+# Working on Windows
+
+jj works the same on all platforms, but there are some caveats that Windows
+users should be aware of.
+
+## Line endings are not converted
+
+jj does not honor `.gitattributes` and does not have a setting like git's
+`core.autocrlf`. This means that line endings will be checked out exactly as
+they are committed and committed exactly as authored. This is true on all
+platforms, but Windows users are most likely to miss CRLF conversion.
+
+## Pagination
+
+[Pagination is disabled by default on
+Windows](https://github.com/martinvonz/jj/issues/2040) because Windows doesn't
+ship with a usable pager.
+
+If you have Git installed, you can use Git's pager and re-enable pagination:
+
+```
```

Comment:
Nit: Missing language for `mkdocs`, I assume it's Powershell.

Reply:
Added the `powershell` language tag, and added the PowerShell prompt `PS>` to all commands, since the shell examples show the `$` prompt.